src/theory/sets/theory_sets_private.cpp

```diff
@@ -1774,13 +814,6 @@ void TheorySetsPrivate::check(Theory::Effort level) {
   Trace("sets-check") << "Sets finish Check effort " << level << std::endl;
 }/* TheorySetsPrivate::check() */
 
-bool TheorySetsPrivate::needsCheckLastEffort() {
```

Comment:
I didn't find any equivalent code for this. Why is this function no longer needed?

Reply:
This function was written for an experimental piece of code that was never integrated (check at LAST_CALL effort in sets was a no-op https://github.com/CVC4/CVC4/pull/3085/files/1a1c83e4667af47eecfe62a779c054e7f6264358#diff-aaa774b6c62e6fbcec9f9c49f7d60a3eL1725); this PR cleans this function up.